lib/javatests/com/google/android/material/button/MaterialButtonToggleGroupTest.java

```diff
@@ -35,31 +35,26 @@
 import org.robolectric.RobolectricTestRunner;
 import org.robolectric.annotation.Config;
 
-/**
- * Tests for {@link com.google.android.material.button.MaterialButtonToggleGroup}.
- */
+/** Tests for {@link com.google.android.material.button.MaterialButtonToggleGroup}. */
 @RunWith(RobolectricTestRunner.class)
 @Config(sdk = 21)
-public class MaterialButtonToggleGroupUnitTest {
+public class MaterialButtonToggleGroupTest {
 
   private static final float CORNER_SIZE = 10f;
   private final Context context = ApplicationProvider.getApplicationContext();
 
   private MaterialButtonToggleGroup toggleGroup;
 
   @Before
-  public void themeApplicationContext() {
+  public void setup() {
```

Comment:
Going to reject this because our readability guidelines suggest splitting the work up as appropriate and giving the methods descriptive names rather than just setUp or tearDown, otherwise pulling in your PR. Thanks for contributing

Reply:
oh interesting thx for the review! :)
I thought there's no way to guarantee the order of execution of multiple `@Before`s, and here don't we want to make sure `context`'s theme has been set before the buttons created?

Reply:
Ohh, good point, will annotate themeApplicationContext with @BeforeClass to guarantee it is called before the other method.

Reply:
It looks like `@BeforeClass` requires the method to be `static`, however the `context` in our `themeApplicationContext()` is non-static.  :/

Reply:
You are right, I changed themeApplicationContext to be a private method called by the other @Before method.

Reply:
yayy thanks!